cmd/dev_common.go

```diff
@@ -142,75 +155,45 @@ func commonCmd(config *devCommonConfig, mode string) error {
 	}
 
 	// Mount the controller
-	destController := os.Getenv("APPSODY_MOUNT_CONTROLLER")
-	if destController != "" {
-		Debug.log("Overriding appsody-controller mount with APPSODY_MOUNT_CONTROLLER env variable: ", destController)
-	} else {
-		// Copy the controller from the installation directory to the home (.appsody)
-		destController = filepath.Join(getHome(config.RootCommandConfig), "appsody-controller")
-		// Debug.log("Attempting to load the controller from ", destController)
-		//if _, err := os.Stat(destController); os.IsNotExist(err) {
-		// Always copy it from the executable dir
-		//Retrieving the path of the binaries appsody and appsody-controller
-		//Debug.log("Didn't find the controller in .appsody - copying from the binary directory...")
-		executable, _ := os.Executable()
-		binaryLocation, err := filepath.Abs(filepath.Dir(executable))
-		Debug.log("Binary location ", binaryLocation)
+	//destController := os.Getenv("APPSODY_MOUNT_CONTROLLER")
+	controllerImageName := overrideControllerImage
+	//if destController != "" {
+	//	Debug.log("Overriding appsody-controller mount with APPSODY_MOUNT_CONTROLLER env variable: ", destController)
+	//} else {
+	//destController := "appsody-controller"
+	controllerVolumeName := fmt.Sprintf("%s-%s", "appsody-controller", CONTROLLERVERSION)
```

Comment:
I'm concerned when the CONTROLLERVERSION is `latest`, the volume named `appsody-controller-latest` will never be updated. 
This should only happen in development scenarios, a.k.a. someone is developing a new controller and wants to test a dev image with the CLI. However, even in that case, I think we need to run the init-controller image to always update the volume.

Reply:
This has been addressed